client/src/pages/OrgPage.tsx

```diff
@@ -107,7 +107,7 @@ export const OrganizationPage = () => {
                             <OrgOverview isActive={key === 'overview'} org={org} setOrg={setOrg} />
                         </Tab.Pane>
                         <Tab.Pane eventKey="members">
-                            <OrgMembers isActive={key === 'members'} />
+                            <OrgMembers isActive={key === 'members'} org={org}/>
```

Comment:
Only members of an org can see the other members of that same org, as per the project spec.
The backend already handles this. The frontend should not allow non-members of org to access the org page of an org they aren't a member of.

You can easily fix this by using `amMemberOfOrg` for the `<Nav.Item>` above relating to Members (just like how we guard Teams and Settings tab items). (I can't highlight the proper lines of code, but it's above these ones in the same file).